Core/src/org/sleuthkit/autopsy/datamodel/BlackboardArtifactNode.java

```diff
@@ -1085,6 +1102,113 @@ public String getItemType() {
+                    ""));
+            sheetSet.put(new NodeProperty<>(
+                    Bundle.BlackboardArtifactNode_createSheet_comment_name(),
+                    Bundle.BlackboardArtifactNode_createSheet_comment_displayName(),
+                    VALUE_LOADING,
+                    ""));
+            if (CentralRepository.isEnabled()) {
+                sheetSet.put(new NodeProperty<>(
+                        Bundle.BlackboardArtifactNode_createSheet_count_name(),
+                        Bundle.BlackboardArtifactNode_createSheet_count_displayName(),
+                        VALUE_LOADING,
+                        ""));
+            }
+            backgroundTasksPool.submit(new GetSCOTask(new WeakReference<>(this), weakListener));
+        }
+
+        sheetSet.put(new NodeProperty<>(
+                Bundle.BlackboardArtifactNode_analysisSheet_sourceType_name(),
+                Bundle.BlackboardArtifactNode_analysisSheet_sourceType_name(),
+                NO_DESCR,
+                getSourceObjType()));
+
+        sheetSet.put(new NodeProperty<>(
+                Bundle.BlackboardArtifactNode_analysisSheet_score_name(),
+                Bundle.BlackboardArtifactNode_analysisSheet_score_name(),
+                NO_DESCR,
+                result.getScore().getSignificance().getDisplayName()));
+
+        sheetSet.put(new NodeProperty<>(
+                Bundle.BlackboardArtifactNode_analysisSheet_conclusion_name(),
+                Bundle.BlackboardArtifactNode_analysisSheet_conclusion_name(),
+                NO_DESCR,
+                result.getConclusion()));
+
+        sheetSet.put(new NodeProperty<>(
+                Bundle.BlackboardArtifactNode_analysisSheet_configuration_name(),
+                Bundle.BlackboardArtifactNode_analysisSheet_configuration_name(),
+                NO_DESCR,
+                result.getConfiguration()));
+
+        sheetSet.put(new NodeProperty<>(
+                Bundle.BlackboardArtifactNode_analysisSheet_justifaction_name(),
+                Bundle.BlackboardArtifactNode_analysisSheet_justifaction_name(),
+                NO_DESCR,
+                result.getJustification()));
+    }
+
+    private String getSourceObjType() {
+        if (srcContent instanceof BlackboardArtifact) {
+            BlackboardArtifact srcArtifact = (BlackboardArtifact) srcContent;
+            try {
+                return srcArtifact.getType().getDisplayName();
+            } catch (TskCoreException ex) {
+                Exceptions.printStackTrace(ex);
+            }
+        } else if (srcContent instanceof Volume) {
+            return "Volumn";
+        } else if (srcContent instanceof AbstractFile) {
+            return "File";
+        } else if (srcContent instanceof Image) {
+            return "Disk Image";
+        } else if (srcContent instanceof VolumeSystem) {
+            return "File";
+        } else if (srcContent instanceof OsAccount) {
+            return "Os Account";
```

Comment:
Should this be "OS Account" to be consistent with [OSAccounts.java](https://github.com/sleuthkit/autopsy/blob/develop/Core/src/org/sleuthkit/autopsy/datamodel/OsAccounts.java#L85)?

Reply:
Yes, OS Account, please.